instalint-http-daemon/src/main/webapp/main.js

```diff
@@ -208,6 +189,9 @@ function updateLocationHash(store) {
 }
 
 function analyze(store) {
+    if (!store) {
+        storedAs = "";
+    }
```

Comment:
After having merged I saw, that restoring examples does not work anymore. It might have been caused by this change...